web/src/components/Incidents/IncidentsPage.jsx

```diff
@@ -2,68 +2,109 @@
 import * as React from 'react';
 import { IncidentsHeader } from './IncidentsHeader/IncidentsHeader';
 import { useSafeFetch } from '../console/utils/safe-fetch-hook';
-import {
-  ListPageFilter,
-  PrometheusEndpoint,
-  useListPageFilter,
-} from '@openshift-console/dynamic-plugin-sdk';
+import { PrometheusEndpoint } from '@openshift-console/dynamic-plugin-sdk';
 import { parsePrometheusDuration } from '../console/utils/datetime';
 import { getPrometheusURL } from '../console/graphs/helpers';
 import { createAlertsQuery } from './api';
 import { useTranslation } from 'react-i18next';
 import {
   Bullseye,
-  Dropdown,
-  DropdownPosition,
-  DropdownToggle,
+  Button,
   Flex,
+  FlexItem,
+  Select,
   Spinner,
+  Toolbar,
+  ToolbarContent,
+  ToolbarFilter,
+  ToolbarItem,
 } from '@patternfly/react-core';
 import { Helmet } from 'react-helmet';
-import { useBoolean } from '../hooks/useBoolean';
-import some from 'lodash-es/some';
-import { dropdownItems } from './consts';
+import { dropdownItems, incidentTypeMenuItems } from './consts';
 import { IncidentsTable } from './IncidentsTable';
 import { getIncidentsTimeRanges, processIncidents } from './processIncidents';
-import { filterIncident } from './utils';
+import {
+  filterIncident,
+  onDeleteGroupIncidentFilterChip,
+  onDeleteIncidentFilterChip,
+} from './utils';
 import { groupAlertsForTable, processAlerts } from './processAlerts';
+import { DropdownToggle, Dropdown } from '@patternfly/react-core/deprecated';
```

Comment:
Could you name these as deprecated like [this](https://github.com/openshift/monitoring-plugin/blob/aae8da3647e2637132e8f9f1475ba83558817243/web/src/components/silence-form.tsx#L16)